contracts/asset-proxy/contracts/src/bridges/DydxBridge.sol

```diff
@@ -81,6 +82,18 @@ contract DydxBridge is
 
         // Run operation. This will revert on failure.
         IDydx(_getDydxAddress()).operate(accounts, actions);
+
+        emit ERC20BridgeTransfer(
```

Comment:
I'm wondering if we should even have these events for the non-DEX bridges. It's pretty awkward with both this and `ChaiBridge`, and I'm not sure it really makes data collection any easier without some sort of rate.

Reply:
I'll check with kroeger.

Reply:
do we have another viable option for collecting the precise trade amounts for those bridges?

Reply:
In practicality, Chai and Dydx orders will have their `toAmount`s match up with those in the `Fill` event. All the other fields will also be visible in the `Fill` event. So I'm leaning towards removing them.

Reply:
I guess the argument for keeping it would be to make tracking (successful) fills from these bridges easier since you won't have to parse the asset data. But I think we already have the infra for that?

Reply:
yea I'm not worried about that, we have infra for parsing asset data in all fills. If fill events will accurately reflect traded amounts in Chai and Dydx bridge trades then I'm fine with removing.